$OpenBSD: patch-src_utils_scripts_denoiser_rules,v 1.1 2011/11/28 15:28:49 dcoppa Exp $

Fix the denoiser not to silence test results from 'make check'
(upstream git commit 239003f1fb2ef3da2efb6764dc8133cb73735aea)

--- src/utils/scripts/denoiser.rules.orig	Sun Mar 27 15:01:14 2011
+++ src/utils/scripts/denoiser.rules	Mon Nov 28 10:40:39 2011
@@ -176,6 +176,13 @@ MAKEFLAGS=-w -s
 		b 0
 	}
 
+	# ship the check target, otherwise test results would be stripped
+	/^check-TESTS$/ {
+		s/.*//
+		x
+		b 0
+	}
+
 	# do some output based on rule target
 	/\.o$/ {
 		i\
